src/components/QRCode.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useMemo } from 'react';
```

Comment:
I sometimes found that it was necessary during RN Development to keep the `React` part but not sure if it errors. If no errors, looks good.

Reply:
since React 17, there's no need to import React. I disabled the rule that marked the missing import as an error.